.github/workflows/schemacode_ci.yml

```
@@ -0,0 +1,64 @@
+name: "schemacode_ci"
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  check_skip:
+    runs-on: ubuntu-latest
+    outputs:
+      skip: ${{ steps.result_step.outputs.ci-skip }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - id: result_step
+        uses: mstachniuk/ci-skip@master
+        with:
+          commit-filter: "[skip ci];[ci skip];[skip github]"
+          commit-filter-separator: ";"
+
+  latest:
+    needs: check_skip
+    if: ${{ needs.check_skip.outputs.skip == 'false' }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+        fail-fast: false
+        matrix:
+            os: ["ubuntu-latest"]
+            python-version: ["3.8"]
```

what about
```suggestion
            python-version: ["3.8", "3.10"]
```
to test against "target" and the "most recent"?  or even better
```suggestion
            python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```
against all supported? (if adding 3.10)